src/dbnode/storage/series/types.go

```diff
@@ -384,6 +396,11 @@ func (s Stats) IncColdWrites() {
 	s.coldWrites.Inc(1)
 }
 
+// RecordEncodersPerBlock records the number of encoders histogram.
+func (s Stats) RecordEncodersPerBlock(num int) {
+	s.encodersPerBlock.RecordValue(float64(num))
```

Comment:
more a question: is this metrics reporting works as you're intending? 

like do we want the ability to understand the distribution of #encoders in use at any point (even if this is delayed, say we emit during a tick)?

This metric is going to be modified only when #encoders changes per series, so it'll be more like a trailing max with double counting in some cases.

Reply:
Just realized that this metric will not work as expected. Since each histogram bucket is a counter, we'll end up counting the total number of active encoders across all blocks...